contracts/src/instances/lseth/LsETHTarget0.sol

```diff
@@ -25,10 +23,10 @@ contract LsETHTarget0 is HyperdriveTarget0, LsETHBase {
 
     /// Getters ///
 
-    /// @notice Returns the instance's name.
-    /// @return The instance's name.
-    function name() external pure override returns (string memory) {
-        _revert(abi.encode(NAME));
+    /// @notice Gets the instance's kind.
```

Comment:
```suggestion
    /// @notice Returns the instance's kind.
```

Reply:
ya some of the getters say "returns" and some say "gets"